src/test/java/uk/gov/crowncommercial/dts/scale/cat/service/ProfileManagementServiceTest.java

```diff
@@ -86,6 +87,12 @@ class ProfileManagementServiceTest {
   @Autowired
   private ProfileManagementService profileManagementService;
 
+  @MockBean
+  private BuyerUserDetailsRepo buyerDetailsRepo;
+
+  @MockBean
+  private EncryptionService encryptionService;
```

Comment:
Was there supposed to be some calls to persist the new buyer user record with the encrypted password in here (when buyer user profile is created)?